sass/components/speakers.scss

```
@@ -0,0 +1,43 @@
+.speakers {
+    min-height: 100px;
+    max-width: 1000px;
+    margin: 0 auto;
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+
+
+}
+
+.speaker-container {
+    margin: auto;
+    text-align: center;
+    background-color: #fff;
+    color: #0a0a2a;
```

```suggestion
    background-color: #272727;
    color: #ffffff;
```

Looks good, thanks @rossturk! 

The only suggestion I do have is to darken the background (and thus lighten the text) as to not make it class as much with the rest of the page.